UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.Common/Logging/EventIds.cs

```diff
@@ -35,7 +36,19 @@ public enum EventIds
         /// <summary>
         /// 900008 - The requested resource responded with Accepted.
         /// </summary>
-        Accepted = 900008
+        Accepted = 900008,
+        /// <summary>
+        /// 900009 - Event hub for exchange set service is healthy.
+        /// </summary>
+        EventHubLoggingIsHealthy = 900009,
+        /// <summary>
+        /// 900010 - Event hub for exchange set service is unhealthy.
+        /// </summary>
+        EventHubLoggingIsUnhealthy = 900010,
+        /// <summary>
+        /// 900011 -  Event data for exchange set service event hub health check.
+        /// </summary>
+        EventHubLoggingEventDataForHealthCheck = 900011
     }
```

Comment:
Change comments as below:
/// 900009 - Event hub for external notification service is healthy.
/// 900010 - Event hub for external notification service is unhealthy.
/// 900011 - Event data for external notification service event hub health check.

Reply:
Changes are done as requested.